lib/elliptic/curve/mont.js

```diff
@@ -42,6 +45,12 @@ function Point(curve, x, z) {
 }
 inherits(Point, Base.BasePoint);
 
+MontCurve.prototype.decodePoint = function decodePoint(bytes, enc) {
+  bytes = utils.toArray(bytes, enc);
+
+  return this.point(bytes, 1);
```

Comment:
haha OK :)

I felt like it's easy to see all in one place, and it's going to be more code overall, with a bunch of repetition machinery. I'll have a crack at it.